### PR TITLE
Improve accordion text

### PR DESCRIPTION
**Pull request by mkondratek**
Merging queue 😅 
image

Rel: #94 

## Test plan
1. use chat and see the new accordion

## The look

https://github.com/sourcegraph/jetbrains/assets/19799111/aa73e8da-5d55-417e-a7d3-437f715f2524

### CODE REVIEW

File: src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt [hunk 1]
```diff
@@ -57,6 +58,34 @@ class ContextFilesPanel(
     add(accordionSection, BorderLayout.CENTER)
   }
 
+  private fun deriveAccordionTitle(contextFileFiles: List<ContextFileFile>): String {
+    val uniqueFiles = mutableSetOf<String>()
+
+    val filteredFiles =
```
Comment (pkukielka) [inline src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt]: You can use `distinctBy`

File: src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt [hunk 2]
```diff
@@ -57,6 +58,34 @@ class ContextFilesPanel(
...
+          if (uniqueFiles.contains(file.uri.toString())) {
+            false
+          } else {
+            uniqueFiles.add(file.uri.toString())
+            true
+          }
+        }
+
+    val prefix = "✨ Context: "
+    val lineCount = contextFileFiles.sumOf { it.range?.delta() ?: 0 }
```
Comment (exigow) [inline src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt]: Readability of this line is problematic at first glance. What's the meaning of "delta"? Proposal: `delta` -> `numberOfLines`?

**Review by exigow — APPROVED** [review]
One minor comment. Looks good :+1:

File: src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt (hunk 2 above)
Comment (pkukielka) [reply src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt]: Or even just `length`, as `range.length` will be quite self explaining.

Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt]: good point!

File: src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt (hunk 1 above)
Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt]: hmmm 🤔 I cannot see it.  how could I use it here?

Comment (pkukielka) [reply src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt]: It's available on kotlin collections:
```
listOf<String>("a", "b", "a").distinctBy { it }
res1: kotlin.collections.List<kotlin.String> = [a, b]
```

Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt]: thanks! done

**Pull request merged**